<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSampleSetting</class>
 <widget class="QWidget" name="QSampleSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1912</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSampleSetting</string>
  </property>
  <widget class="QComboBox" name="sampleTubes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>260</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sampleType">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>260</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save_Button">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>812</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="buttons" stdset="0">
    <string>flat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>默认样本管</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>默认样本类型</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
